src/views/CreateGroup/components/CreateGroupForm/formInitialValues.ts

```diff
@@ -0,0 +1,44 @@
+export const initialValues = {
+	groupName: '',
+	isPublic: true,
+	password: '',
+	createMonkey: true,
+	allowGamePredictions: true,
+	twoBeatsMode: true,
+	uploadedImage: '',
+};
+
+interface SettingValue {
+	title: string;
+	explanationTitle: string;
+	explanation: string;
+	key: 'createMonkey' | 'allowGamePredictions' | 'twoBeatsMode';
+}
+
+export const settingValues: SettingValue[] = [
+	{
+		title: 'Create monkey prediction',
+		explanationTitle:
+			'Will add a "monkey" member into your group with a random predictions',
+		explanation:
+			'The monkey will compete with you and your group members during the tournament and you can remove him through the group panel.',
```

Comment:
'The monkey will compete with you and your group members during the tournament. You can remove it anytime through the group admin panel.'